practica2023/src/main/java/com/endava/practica2023/Repository/OrdersRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.endava.practica2023.Repository;
+
+import com.endava.practica2023.Domain.Orders;
+import com.endava.practica2023.Domain.TicketCategory;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface OrdersRepository extends JpaRepository<Orders, Integer> {
+    List<Orders> findOrdersByTicketCategory(TicketCategory ticket_category_id);
```

Comment:
Naming convention in Java is camel case for methods and variables, not how you name them with _ between words